kerosene/training/trainers.py

```diff
@@ -40,21 +41,27 @@ class ModelTrainer(ApexModule):
 
     def __init__(self, model_name, model, criterion, optimizer, scheduler, metric_computer: Metric):
         super().__init__(model, optimizer)
+        self._status = Status.INITIALIZING
+
         self._model_name = model_name
 
         self._criterion = criterion
         self._scheduler = scheduler
         self._metric_computer = metric_computer
 
-        self._step_train_loss = torch.tensor([0.0])
-        self._step_valid_loss = torch.tensor([0.0])
-        self._step_train_metric = torch.tensor([0.0])
-        self._step_valid_metric = torch.tensor([0.0])
+        self._step_train_loss = torch.Tensor().new_zeros((1,), dtype=torch.float32, device="cpu")
```

Comment:
PyTorch doc says: "To create a tensor with pre-existing data, use torch.tensor()."
and
"To create a tensor with similar type but different size as another tensor, use tensor.new_* creation ops"
But we don't really have pre-existing data here. So using the `new_*` interface.